### PR TITLE
Update DSM7.0 Toolchains

### PR DESCRIPTION
**Pull request by publicarray**
* Newer GCC compiler

### CODE REVIEW

File: mk/spksrc.tc-vers.mk [hunk 1]
```diff
@@ -64,4 +64,5 @@ endif
 
 ifeq ($(TC_VERS),7.0)
 TC_BUILD = 40000
```
Comment (hgy59) [inline mk/spksrc.tc-vers.mk:66]: we could now set the build version to 41890

File: mk/spksrc.tc-vers.mk [hunk 2]
```diff
@@ -64,4 +64,5 @@ endif
...
+TC_VERS = 7.0.0
```
Comment (hgy59) [inline mk/spksrc.tc-vers.mk:67]: We might get unexpected side effects, as the DSM version is still 7.0 and not 7.0.0

**Comment by publicarray** [comment]
FYI  DSM7.0 is still missing the GPL source code https://sourceforge.net/projects/dsgpl/files/Synology%20NAS%20GPL%20Source/

**Comment by publicarray** [comment]
@hgy59 I noticed that Synology is also hosting them: https://archive.synology.com/download/ToolChain/toolchain/7.0-41890
Maybe we can add support for multiple mirrors?

**Comment by publicarray** [comment]
Remember to do a `make dist-clean`

**Pull request merged**